nix/tools.nix

```diff
@@ -0,0 +1,14 @@
+final: prev: {
+  hacking-on-cachix-shell = final.haskellPackages.shellFor {
+    withHoogle = true;
+    packages = hpkgs: [  ];
+    nativeBuildInputs = [
```

Comment:
This should be deduplicated with the list in `flake.nix`

Reply:
removed in https://github.com/cachix/cachix/pull/434/commits/94aeb8b22da627a844bbb1d0a95205fa524da593